--- linuxrc.syno.impl	2024-01-31 10:10:29.000000000 +0800
+++ /home/yxl/my_proj/syno/analysis_oec_dsm/reextract/from_img_initrd/root/linuxrc.syno.impl	2025-04-16 19:27:11.000000000 +0800
@@ -292,7 +292,7 @@
 	insmod /lib/modules/of_i2c.ko
 	insmod /lib/modules/i2c-mpc.ko
 fi
-
+sleep 5
 if [ -x /usr/syno/bin/synocfgen ]; then
        insmod /lib/modules/synobios.*
        /bin/mknod /dev/synobios c 201 0
@@ -300,7 +300,7 @@
         /usr/syno/bin/synocfgen
         RetCfg=$?
         echo "/usr/syno/bin/synocfgen returns $RetCfg"
         # shellcheck disable=SC2046 # Because delimiter of rmmod arguments is spaces
-        rmmod `/sbin/lsmod | /usr/bin/cut -f 1 -d ' ' | /bin/grep synobios`
+        # rmmod `/sbin/lsmod | /usr/bin/cut -f 1 -d ' ' | /bin/grep synobios`
 		if [ "ppc853x" = "$UniqueRD" ]; then
 			rmmod i2c_mpc
 			rmmod of_i2c
