--- etc/rc	2024-01-31 10:10:29.000000000 +0800
+++ etc/rc	2026-06-09 00:00:00.000000000 +0800
@@ -236,7 +236,7 @@
 	/usr/syno/bin/syno_microp_control STATUS_GREEN_LED_BLINKING
 
 	# shellcheck disable=SC2046 # Because delimiter of rmmod arguments is spaces
-	rmmod `/sbin/lsmod | /usr/bin/cut -f 1 -d ' ' | /bin/grep synobios`
+	# rmmod `/sbin/lsmod | /usr/bin/cut -f 1 -d ' ' | /bin/grep synobios`
 fi
 
 if [ $KERNEL_VCODE -ge "$(KernelVersionCode "3.4")" ]; then
@@ -510,6 +510,7 @@
 echo "============ Date ============"
 date
 echo "=============================="
+/sbin/getty 1500000 console
 
 exit 0
