--- etc/passwd	2025-06-17 14:29:39.000000000 +0800
+++ etc/passwd	2026-06-09 00:00:00.000000000 +0800
@@ -1,4 +1,4 @@
-root:x:0:0::/root:/bin/ash
+root:$1$hA3RciZk$gLdumik0d/FLPR8sPUFSX/:0:0::/root:/bin/ash
 system:x:1:1::/usr/syno/synoman:/usr/bin/nologin
 daemon:x:2:2::/:/bin/sh
 SYSTEM_ADMIN:x:3:101::/nonexist:/usr/bin/nologin
--- etc/shadow	2025-06-17 14:29:39.000000000 +0800
+++ etc/shadow	2026-06-09 00:00:00.000000000 +0800
@@ -1,4 +1,4 @@
-root:*:10933:0:99999:7:::
+root:$1$hA3RciZk$gLdumik0d/FLPR8sPUFSX/:10933:0:99999:7:::
 admin:$1$synergy$O31nVEgYyKUdmjDRPUq8p.:10933:0:99999:7:::
 guest:$1$synergy$O31nVEgYyKUdmjDRPUq8p.:10933:0:99999:7::1:
 system:*:10933:0:99999:7:::
